src/site-apps.ts

```diff
@@ -101,17 +114,19 @@ export class SiteApps {
                         }
                     })
                 }
-            } else if (response['siteApps'].length > 0) {
+            } else if (this._decideServerResponse.length > 0) {
                 logger.error('PostHog site apps are disabled. Enable the "opt_in_site_apps" config to proceed.')
                 this.loaded = true
             } else {
                 this.loaded = true
             }
         } else {
             this.loaded = true
-            this.enabled = false
         }
     }
 
-    // TODO: opting out of stuff should disable this
+    afterDecideResponse(response: DecideResponse): void {
+        this._decideServerResponse = response['siteApps']
```

Comment:
The variable `_decideServerResponse` makes me think this contains the whole response, not just the `siteApps` key

Reply:
renamed to "_decideServerSiteAppsResponse"